src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -66,14 +72,18 @@ public class LinkTagHelper : TagHelper
         private enum Mode
         {
             /// <summary>
+            /// Adding file version to the generated urls.
```

Comment:
previous description was more correct i.e. use "Just adding a file version for the src." because modes overlap and could add file versions in other modes.